frontend/tests/frontend.spec.js

```diff
@@ -0,0 +1,160 @@
+const { test, expect, _electron: electron } = require("@playwright/test");
+
+let electronApp;
+let window;
+
+test.beforeAll(async () => {
+  // Launch app
+  electronApp = await electron.launch({ args: ["."] });
+
+  // Wait for the first BrowserWindow to open and return its Page object
+  window = await electronApp.firstWindow();
+
+  // Take initial screenshot
+  await window.screenshot({ path: "tests/screenshots/intro.png" });
+});
+
+test.afterAll(async () => {
+  await window.screenshot({ path: "tests/screenshots/final.png" });
+  await window.context().close();
+  await window.close();
+  await electronApp.close();
+});
+
+test("start app", async () => {
+  const isPackaged = await electronApp.evaluate(async ({ app }) => {
+    // This runs in Electron's main process, parameter here is always
+    // the result of the require('electron') in the main app script.
+    return app.isPackaged;
+  });
+
+  expect(isPackaged).toBe(false);
+
+  // Evaluation expression in the Electron context.
+  const appPath = await electronApp.evaluate(async ({ app }) => {
+    // This runs in the main Electron process, parameter here is always
+    // the result of the require('electron') in the main app script.
+    return app.getAppPath();
+  });
+
+  console.log(appPath);
```

Comment:
I think we can remove this to keep the output a little cleaner since it breaks up the test logging a bit